analysis.py

```diff
@@ -23,6 +25,53 @@
 
 
 LOGGER_DATE_FMT = "%Y-%m-%d %H:%M:%S.%f"
+REPORT_STUB = {
+    "text": "Example message",
+}
+
+
+REPORT_HOOK_URL = "https://chat.brainbot.com/hooks/"
+
+
+DEFAULT_INCLUDE = "^Transfer.*|.*ChannelTask$|DepositTask$|.*MS.*|.*PFS.*"
+
+
+def filter_report(
+    report: List[Dict[str, Any]], include: str = DEFAULT_INCLUDE,
+) -> List[Dict[str, Any]]:
+    expression = re.compile(include)
+
+    def flt(e: Dict[str, Any]) -> Optional[Match[str]]:
+        return expression.match(e["name"])
+
+    return list(filter(flt, report,))
+
+
+def post_report(report: List[Dict[str, Any]], url: str) -> None:
+    message = REPORT_STUB.copy()
+    message["text"] = json_list_to_md_table(filter_report(report))
+    requests.post(url, json=message)
+
+
+def safe_format_number(value: Any) -> str:
+    result = value
+    try:
+        if "." in value:
+            result = f"{float(value):.04}"
+    except ValueError:
+        pass
+    finally:
+        return str(result)
```

Comment:
Is this different from
```suggestion
    return str(result)
```
?